lexicons/place/stream/default/metadata.json

```diff
@@ -0,0 +1,166 @@
+{
+  "lexicon": 1,
+  "id": "place.stream.default.metadata",
```

Comment:
Perhaps `place.stream.metadata.configuration`? To reflect this is the input metadata instead of the output.